src/utils/Options.ts

```diff
@@ -104,56 +76,74 @@ module n3Charts.Utils {
 
     sanitizeAxes(axes: any) {
       // Map object keys and return a new object
-      return <IAxes> Object.keys(axes).reduce((prev, key) => {
+      return <IAxesSet> Object.keys(axes).reduce((prev, key) => {
         prev[key] = new AxisOptions(axes[key]);
         return prev;
       }, {});
     }
 
-    sanitizeMargin(margin) {
-      return <IMargin> {
-        top: parseFloat(margin.top),
-        left: parseFloat(margin.left),
-        bottom: parseFloat(margin.bottom),
-        right: parseFloat(margin.right)
-      };
-    }
-
-    isValidAxisSide(side:string): Boolean {
-      return d3.values(Factory.Axis.SIDE).indexOf(side) === -1
-        ? false : true;
-    }
-
-    isValidSeriesType(type:string): Boolean {
-      return d3.values(Options.SERIES_TYPES).indexOf(type) === -1
-        ? false : true;
-    }
-
     getAbsKey(): string {
-      if (!this.axes[Factory.Axis.SIDE.X]) {
-        throw new TypeError('Cannot find abs key : ' + Factory.Axis.SIDE.X);
+      if (!this.axes[AxisOptions.SIDE.X]) {
+        throw new TypeError('Cannot find abs key : ' + AxisOptions.SIDE.X);
       }
 
-      return this.axes[Factory.Axis.SIDE.X].key;
+      return this.axes[AxisOptions.SIDE.X].key;
     }
 
     getByAxisSide(side: string) {
-        if (!this.isValidAxisSide(side)) {
+        if (!AxisOptions.isValidSide(side)) {
             throw new TypeError('Cannot get axis side : ' + side);
         }
 
         return this.axes[side];
     }
 
-    getSeriesByType(type: string): SeriesOptions[] {
-      if (!this.isValidSeriesType(type)) {
+    getSeriesByType(type: string) {
+      if (!SeriesOptions.isValidType(type)) {
         throw new TypeError('Unknown series type: ' + type);
       }
 
-      return this.series.filter((series) => series.type.indexOf(type) !== -1);
+      return this.series.filter((s) => s.hasType(type));
+    }
+
+    static getBoolean(val: any, def: boolean = true) {
```

Comment:
I'm not fan of abbreviating variables, especially in methods' signature... Do you mind changing this ?